src/k8s-dashboard/deploy/start.sh.template

```diff
@@ -20,8 +20,15 @@
 pushd $(dirname "$0") > /dev/null
 
 {% if cluster_cfg['cluster']['common']['k8s-rbac'] != 'false' %}
+mkdir -p certs
+cp {{ cluster_cfg["pylon"]["ssl"]["crt_path"] }} certs/tls.cert
```

Comment:
Do we need this key and cert? Because I found you added the option ```auto-generate-certificates``` and it seems dashboard will generated a selfsign cert?

Reply:
To access k8s-dashboard directly without kubectl proxy, we need set certs file. And `auto-generate-certificates` only used as a fallback.

According to https://github.com/kubernetes/dashboard/blob/master/docs/user/installation.md#recommended-setup
```
Custom certificates have to be stored in a secret named kubernetes-dashboard-certs in the 
same namespace as Kubernetes Dashboard. Assuming that you have tls.crt and tls.key files 
stored under $HOME/certs directory, you should create secret with contents of these files:

kubectl create secret generic kubernetes-dashboard-certs --from-file=$HOME/certs -n 
kubernetes-dashboard

For Dashboard to pickup the certificates, you must pass arguments --tls-cert-file=/tls.crt and
 --tls-key-file=/tls.key to the container.
```
k8s dashboard will read the certs under /certs folder